src/app/templates/index.ftl

```diff
@@ -149,6 +149,10 @@
                 "${pinnedReport.name}",
                 </#list>];
             </#if>
+            var runReportBtn = document.querySelector('#run-report')
+            runReportBtn.addEventListener('click', () => {
+                window.sessionStorage.removeItem("selectedRunningReportTab")
+            })
```

Comment:
Is it possible to test session storage in `IndexPageTests.kt`?

Reply:
Also, I'll need mrc-2150 to be merged first so I can make some final tweaks to this ticket.